packages/atlas/src/providers/videoWorkspace/hooks.ts

```diff
@@ -45,7 +45,7 @@ export const useVideoWorkspaceData = () => {
       },
     }
   )
-  const { categories } = useCategories()
+  const { categories } = useQnCategories()
```

Comment:
We discussed this with Bedeho yesterday - what kind of categories we should use during publishing. We decided that the select in video workspace should show display categories. Also, in category config, for each display category, aside list of `videoCategories`, we define `defaultVideoCategory` which will be used when user tries to publish video under this display category.